main/src/library/List.flix

```diff
@@ -880,6 +880,11 @@ namespace List {
     pub def toString(xs: List[Char]): Str =
         List.foldLeft((acc, c) -> String.concat(acc, Char.toString(c)), "", xs)
 
+    ///
+    /// Returns `xs` as a mutable list.
+    ///
+    pub def toMutable(xs: List[a]): MutList[a] & Impure =
```

Comment:
Just `toMut`.

Reply:
Just double checking - the equivalent operations in `Set` and `Map` are named `toMutable`, so if we do `toMut` here, then we should probably rename the others for consistency?

Reply:
Yes, let us rename those too.